stdlib/README.md

```diff
@@ -0,0 +1,3 @@
+`libc.ll` is generated from a modified [musl](http://www.etalabs.net/musl/).
```

Comment:
This document is the LLVM standard library. I think I can just remove this document now because it is used when we are going to define the LLVM semantics (not syntax).